script.js

```diff
@@ -0,0 +1,126 @@
+//1
+console.log("hello world");//english
+console.log("merhaba dunya");//turkish
+
+//2
+console.log('I\'m awesome');
+
+//3
+
+let age;
+console.log("i was happy when i was 14");
+console.log("14");
+age=14;
+console.log("i was happy when i was 14");
+console.log("14");
+
+//4
+
+let a ="sky"
+console.log("value of string wil be:rainbow");
+console.log(a);
+a="rainbow"
+console.log("value of string will be:rainbow");
+console.log(a);
+
+//5
+
+let number = 7.25;
+console.log(number); 
+let roundedNumber = Math.round(number);
+console.log(roundedNumber); 
+
+if (number > roundedNumber){
+   console.log(number);
+   
+}
+
+else{ 
+  console.log(roundedNumber);
+}
+
+//6 arrays 
+
+let animals =[];
+console.log("my value of arrays is animals");
+console.log(animals);
+
+let myFavoriteAnimals =["tiger", "koala", "raccoon"];
+console.log(myFavoriteAnimals);
+
+myFavoriteAnimals.push('kitten');
+console.log(myFavoriteAnimals);
+
+//7
+
+let myString = "this is a test";
+console.log(myString);
+console.log(myString.length);
+
+//8
+
+x = 7  
+x = x %3
+console.log(x);
+
+y = 9
+y = y %3
+console.log('remainer is 0 from 9/3');
+
+z = 19
+z = z %4
+console.log('remainer is 3 from 19/4');
+
+q = 25
+q = q %7
+console.log('remainer is 4 from 25/7');
+
+//9
+
+let myFavoriteApples =["yellow", "green", 1000];
+console.log(myFavoriteApples);
+
+console.log(6/0===10/0);
+console.log('3 equals compare two values if they are same values and types.');
+
+
+//10
+const grid = [[0, 1], [1, 1], [2, 1], [3, 1]];
+console.log(grid[2][1]);
+
+//11
+
+let counter = 0;
+console.log('first method');
+console.log(counter + 2);
+
+
+
+console.log('second method');
```

Comment:
The second method to increment counter by 2 should be:
`counter++;`
`counter++;`

`counter = 2` assigns the value of 2 to counter, but doesn't increase the value by 2.